### PR TITLE
Assessments/unit 01 numeric

### PR DESCRIPTION
**Pull request by theoranelletti**
I don't understand how I didn't get any of the first 4 right.

### CODE REVIEW

**Comment by theoranelletti** [comment]
I forgot to request reviewer from a while ago

File: assessments/unit-01-numeric/code.js [hunk 1]
```diff
@@ -7,3 +7,43 @@ const G = 6.6743e-11;
 // syntactically correct function skeleton, more credit for a skeleton with a
 // reasonable argument list, yet more credit for a working function, and full
 // credit for a clear and simple working function.
+
+const itemsLeftOver = (numpeople, numitems) => {
+  return
+};
+
+const areaOfCircles = (r) => {
+  return Math.PI(r ** 2);
```
Comment (gigamonkey) [inline assessments/unit-01-numeric/code.js:16]: I don't know if you were trying to treat `Math.PI` as a function (what Javascript will think you're doing) or maybe just using "putting two things next to each other to mean multiplication" they way you can in math. At any rate, neither one quite works. You need `Math.PI * r ** 2`.

File: assessments/unit-01-numeric/code.js [hunk 2]
```diff
@@ -7,3 +7,43 @@ const G = 6.6743e-11;
...
+  return length ** 3;
+};
+
+const populationGrowth = (current, growth) => {
+  return current * growth;
+};
+
+const earnedRunAverage = (earnedruns, innings) => {
+  return (earnedruns / innings) * 9;
+};
```
Comment (gigamonkey) [inline assessments/unit-01-numeric/code.js:29]: These are all right.

**Review by gigamonkey — APPROVED** [review]
I assume this is mostly here to clear the Github decks. You did indeed get several of the first four right. Anyway, for Github purposes, you should go ahead and merge this.

**Pull request merged**